test/library/alias.lua

```diff
@@ -0,0 +1,11 @@
+-- Simple tests to ensure the alias functions work as expected
+function test.aliases()
+    expect.eq(false, dfhack.internal.isAlias("foo"))
```

Comment:
this can actually be flaky since it depends on the current active environment. these function tests allow you to specify a setup/teardown wrapper where you can save the current aliases, clear the alias list for the test, and restore the original aliases afterwards. See the orders plugin tests for an example.